transform-message-formats/tests/test.bal

```diff
@@ -0,0 +1,187 @@
+import ballerina/http;
+import ballerina/test;
+
+final http:Client cl = check new (string `http://localhost:${port}/healthcare`);
+
+const GRAND_OAKS_HOSPITAL = "grand oak community hospital";
```

Comment:
OAK vs OAKS? Can't we use ID here.

Reply:
It's oak https://github.com/ballerina-guides/integration-tutorials/blob/2d5621d772e988edb9a8d7fe1d1d972ffd377415/backends/hospital-service/main.bal#L90

We need to have this to mock the implementation of the back end

Reply:
I meant the const name is wrong.

Reply:
oh. Nice catch!